src/XrdHttp/XrdHttpProtocol.cc

```diff
@@ -1135,6 +1139,15 @@ int XrdHttpProtocol::Config(const char *ConfigFN, XrdOucEnv *myEnv) {
 //
    if (myEnv->Get("XrdCache")) hasCache = true;
 
+   if(strlen(xrdcorsLibPath) && LoadCorsHandler(&eDest, xrdcorsLibPath)) {
+     return 1;
+   }
+
+  // If xrdcors was loaded, configure it
+  if(xrdcors) {
```

Comment:
This section can be combined with the previous one. They are both codependent and either both should occur or none should occur. So, if LoadCorsHandler() succeeds then one should continue to config it. Separate tests are unnecessary.